templates/contrib/pc/rhel6.tpl.yaml

```diff
@@ -0,0 +1,157 @@
+{% set diskbus = diskbus |default("virtio") %}
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: {{ os }}-{{ item.workload }}-{{ item.flavor }}
```

Comment:
I think you should change name of this template, because if applied, it will replace regular rhel 6 template in cluster.
and currently this template is not tested. Can you please update tests here: https://github.com/kubevirt/common-templates/blob/master/automation/test-linux.sh

Reply:
Yes, let's just append `-pc` to the name

Reply:
That's done ;)